plugins/gcp/resoto_plugin_gcp/resources/compute.py

```diff
@@ -3291,6 +3291,7 @@ def collect_individual(cls: Type[GcpResource], builder: GraphBuilder, zone: str,
             zone=zone,
             machineType=name,
         )
+        result[InternalZoneProp] = zone
```

Comment:
Please add a comment explaining this line of code. It is not obvious.

Reply:
will do